core/model/src/main/java/com/looker/core/model/Repository.kt

```diff
@@ -377,6 +377,18 @@ data class Repository(
                 name = "SimpleX Chat F-Droid",
                 description = "SimpleX Chat official F-Droid repository.",
                 fingerprint = "9F358FF284D1F71656A2BFAF0E005DEAE6AA14143720E089F11FF2DDCFEB01BA"
+            ),
+            defaultRepository(
```

Comment:
You need to add them to `newlyAdded` in line 365 and also update version [here](https://github.com/Droid-ify/client/blob/80aee42fa96a03e86340a7dea8884d555a6a076d/app/src/main/kotlin/com/looker/droidify/database/Database.kt#L186)